.github/workflows/windows-xp-clang.yml

```diff
@@ -0,0 +1,62 @@
+name: windows-xp-clang
+on:
+  push:
+    branches: [master]
+    paths:
+    - '.github/workflows/windows-xp-clang.yml'
+    - 'CMakeLists.txt'
+    - 'cmake/**'
+    - 'src/*'
+    - 'src/layer/*'
+    - 'src/layer/x86/**'
+    - 'src/layer/vulkan/**'
```

Comment:
If vulkan support is disabled anyway, do not trigger ci for changes in vulkan subdir